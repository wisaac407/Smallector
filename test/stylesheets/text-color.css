/*
 * commands: split
 */

.red-text {
  color: red;
}

.green-text {
  color: green;
}

.blue-text {
  color: blue;
}