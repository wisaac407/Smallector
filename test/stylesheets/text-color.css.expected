/*
 * commands: split
 */

.d-a {
  color: red;
}

.c-a {
  color: green;
}

.b-a {
  color: blue;
}